File: src/main/webapp/js/model/order.js
function doSearch() {
    $('#tt').datagrid('load',{
        orderNum:$('#orderNum').val(),
        consignorNum:$('#consignorNum').val(),
        providerNum:$('#providerNum').val(),
        gmtCreat:$('#gmtCreat').val(),
        deliveryDate:$('#deliveryDate').val(),
    });
};
$(function () {
    $('#tt').datagrid({
        columns: [[
            {field: 'id', title: 'id', width: 100},
            {field: 'orderNum', title: '采购单号', width: 100},
            {field: 'consignorNum', title: '货主编号', width: 100},
            {field: 'consignorName', title: '货主名字', width: 100},
            {field: 'providerNum', title: '供应商编号', width: 100},
            {field: 'providerName', title: '供应商名字', width: 100},
            {field: 'totalNum', title: '总数', width: 100, editor: 'numberbox'},
            {field: 'totalPrice', title: '总价', width: 100},
            {field: 'gmtCreat', title: '提交时间', width: 100},
            {field: 'deliveryDate', title: '预计到货日期', width: 100,}
        ]],
        url: '/order/search',
        singleSelect: true
    });
});

File: src/main/java/cn/test/pojo/Order.java
package cn.test.pojo;

import java.math.BigDecimal;
import java.util.Date;

public class Order {
    private Long id;

    private Long orderNum;

    private Date deliveryDate;

    private String consignorNum;

    private String consignorName;

    private String providerNum;

    private String providerName;

    private Long totalNum;

    private BigDecimal totalPrice;

    private Date gmtCreat;

    private Date gmtModified;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public Long getOrderNum() {
        return orderNum;
    }

    public void setOrderNum(Long orderNum) {
        this.orderNum = orderNum;
    }

    public Date getDeliveryDate() {
        return deliveryDate;
    }

    public void setDeliveryDate(Date deliveryDate) {
        this.deliveryDate = deliveryDate;
    }

    public String getConsignorNum() {
        return consignorNum;
    }

    public void setConsignorNum(String consignorNum) {
        this.consignorNum = consignorNum == null ? null : consignorNum.trim();
    }

    public String getConsignorName() {
        return consignorName;
    }

    public void setConsignorName(String consignorName) {
        this.consignorName = consignorName == null ? null : consignorName.trim();
    }

    public String getProviderNum() {
        return providerNum;
    }

    @Override
    public String toString() {
        return "Order{" +
                "id=" + id +
                ", orderNum=" + orderNum +
                ", deliveryDate=" + deliveryDate +
                ", consignorNum='" + consignorNum + '\'' +
                ", consignorName='" + consignorName + '\'' +
                ", providerNum='" + providerNum + '\'' +
                ", providerName='" + providerName + '\'' +
                ", totalNum=" + totalNum +
                ", totalPrice=" + totalPrice +
                ", gmtCreat=" + gmtCreat +
                ", gmtModified=" + gmtModified +
                '}';
    }

    public void setProviderNum(String providerNum) {
        this.providerNum = providerNum == null ? null : providerNum.trim();
    }

    public String getProviderName() {
        return providerName;
    }

    public void setProviderName(String providerName) {
        this.providerName = providerName == null ? null : providerName.trim();
    }

    public Long getTotalNum() {
        return totalNum;
    }

    public void setTotalNum(Long totalNum) {
        this.totalNum = totalNum;
    }

    public BigDecimal getTotalPrice() {
        return totalPrice;
    }

    public void setTotalPrice(BigDecimal totalPrice) {
        this.totalPrice = totalPrice;
    }

    public Date getGmtCreat() {
        return gmtCreat;
    }

    public void setGmtCreat(Date gmtCreat) {
        this.gmtCreat = gmtCreat;
    }

    public Date getGmtModified() {
        return gmtModified;
    }

    public void setGmtModified(Date gmtModified) {
        this.gmtModified = gmtModified;
    }
}